Client/Frontend/ClientPreferences.swift

```diff
@@ -111,8 +111,8 @@ extension Preferences {
       Option<Bool>(key: "general.default-browser-callout-dismissed", default: false)
 
     /// Whether or not the app (in regular browsing mode) will follow universal links
-    static let followUniversalLinks = Option<Bool>(key: "general.follow-universal-links", default: true)
-
+    static let followUniversalLinks = Option<Bool>(key: "general.follow-universal-links", default: false)
+    
```

Comment:
I think you still want this to be `true` by default

Reply:
No we want to swap the default value to false for new users, I added the ticket https://github.com/brave/brave-ios/issues/5896 here in this Pr like I mentioned in previous comment.

Reply:
We have to be careful here.
No changes for existing users.
For new users we want them to stay in Brave by default.

Is this change not going to affect existing users?

Reply:
No it will not effect existing users, this is the default value of that preference. And guys there is literally no other way to do this either this or previous version with new preference and migration.

Reply:
The default value of the pref is what you get when it hasn't been set before, so it will effect existing users who have never touched the option in the settings page.

So just to clarify this isn't just a copy change? We actually want to alter the default behaviour of the WKWebView to not open universal links?

Reply:
The default value change is reversed and change for first time users added in app launch inside migrate elements.
Fix at https://github.com/brave/brave-ios/pull/5890/commits/eb864ba9df71067614d51ea01df2ea1c739d4b91

Reply:
yeah this is what I would do tbh, no changes to the setting and set it for first users
@kylehickinson and yes to your question